js/src/SmartConnect/index.js

```diff
@@ -134,8 +134,9 @@ function smartConnect(publicAPI, model) {
     return session;
   };
 
-  function cleanUp() {
+  function cleanUp(timeout) {
     if (session) {
+      timeout > 0 && session.call('application.exit.later', [timeout]);
```

Comment:
Can you rewrite that as a if?

```js
if (timeout > 0) {
  session.call('application.exit.later', [timeout])
}
```

Reply:
Also can you rework your commit (maybe via force push) to be 

`fix(SmartConnect): Allow delete to call exit later`

Reply:
ok, done